src/modules/SourceDialogs.rb

```diff
@@ -2540,7 +2540,11 @@ def EditDisplayInt(repository)
       Builtins.y2milestone("Ret: %1", ret)
       UI.CloseDialog
 
-      ret == :ok ? GetURL() : ""
+      if ret == :ok || ret == :next
```

Comment:
Maybe you can change this by

```ruby
[:ok, :next].include?(ret) ? GetURL() : ""
```
to make Rubocop happy

Reply:
I mean, to use less lines and keep the class below the 2000 lines.

Reply:
or even

```ruby
return GetURL() if ret == :ok || ret == :next

""
```

:see_no_evil:

Reply:
I like the first one :-)